src/main/java/uk/gov/companieshouse/insolvency/delta/mapper/InsolvencyMapper.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.companieshouse.insolvency.delta.mapper;
+
+import org.mapstruct.Context;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import uk.gov.companieshouse.api.delta.Insolvency;
+import uk.gov.companieshouse.api.insolvency.InternalCompanyInsolvency;
+
+@Mapper(componentModel = "spring", uses = {CaseMapper.class})
+public interface InsolvencyMapper {
+
+    @Mapping(target = "etag", ignore = true) // doesn't exist on source
+    @Mapping(target = "status", ignore = true) // doesn't exist on source
+    @Mapping(target = "deltaAt", source = "deltaAt")
```

Comment:
deltaAt maynot need to be explicitly mentioned as both target and source are same?

Reply:
We have decided to leave it because it makes the overall structure more transparent and it becomes immediately obvious what object we are mapping from.